.goreleaser.yaml

```diff
@@ -0,0 +1,86 @@
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - main: ./cmd/sprout/
+    id: "sprout"
+    binary: sprout
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - 386
+      - arm64
+      - arm
+  - main: ./cmd/farmer/
+    id: "farmer"
+    binary: farmer
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - 386
+      - arm64
+      - arm
+  - main: ./cmd/grlx/
+    id: "grlx-cli"
+    binary: grlx-cli
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - amd64
+      - 386
+      - arm64
+      - arm
+    ldflags:
+      - -X main.GitCommit={{.ShortCommit}} -X main.Tag=v{{.Version}}
+
+# generates macOS universal binaries
+universal_binaries:
+  - id: grlx-universal
```

Comment:
can we call this grlx-macos-universal

Reply:
This is added in the newest commit to create the binary to be named `grlx-cli-darwin-universal`